tasks/add-missing-file-checksums/src/index.ts

```diff
@@ -1,8 +1,9 @@
-// @ts-expect-error
 import { runCumulusTask } from '@cumulus/cumulus-message-adapter-js';
 import * as awsClients from '@cumulus/aws-client/services';
 import * as S3 from '@cumulus/aws-client/S3';
-import { Granule, GranuleFile, HandlerEvent, HandlerInput } from './types';
+import { Context } from 'aws-lambda';
```

Comment:
This set of changes is the only non-version related change - adding explicit types was needed to interface with cma-js.